### ISSUE TITLE
fail to run on macOS 10.15

### ISSUE THREAD

**Issue by billypchan** [peripheryapp/periphery#81]
Hi! After update to macOS 10.15 I find periphery can not run for this error:

```
% periphery 
dyld: Library not loaded: @rpath/lib_InternalSwiftSyntaxParser.dylib
  Referenced from: /usr/local/bin/periphery
  Reason: image not found
zsh: abort      periphery
```

I tried reinstall periphery but still no luck. Any suggestion? Thanks!

**Comment by Dfmarulanda** [comment]
Same here

**Comment by ileitch** [comment]
I've just released 1.7.1, can you please give it a try?

**Comment by billypchan** [comment]
@ileitch it works now! thanks!
